reframe/frontend/cli.py

```diff
@@ -5,9 +5,10 @@
 import reframe
 import reframe.core.logging as logging
 import reframe.utility.os as os_ext
-from reframe.core.exceptions import (EnvironError, ReframeError,
+from reframe.core.exceptions import (EnvironError, ReframeError, ConfigError,
```

Comment:
Please have a look at the [import guidelines](https://github.com/eth-cscs/reframe/wiki/Coding-Style-Guide#imports).